Makefile

```diff
@@ -9,3 +13,19 @@ check:
 
 test: 
 	Rscript  -e "library(testthat); test()"
+
+dev_image:
+	@docker run -it --name irkernel-build \
+	$(DEV_IMAGE) bash -c 'Rscript -e "install.packages(c('\''devtools'\''), repos = c('\''http://irkernel.github.io/'\'', '\''http://cran.rstudio.com'\''))" && \
+	                        Rscript -e "library(devtools)" \
+	                        -e "install_github('\''IRkernel/repr'\'')" \
+	                        -e "install_github('\''IRkernel/IRdisplay'\'')"'
+	@docker commit irkernel-build $(DEV_IMAGE)
```

Comment:
Shouldn't this be committing with a different name, not overwriting `jupyter/r-notebook`? The user should still be able to run the stable version after they've run this, without the two interfering.